src/integrant/repl.clj

```diff
@@ -16,8 +16,16 @@
     (do (alter-var-root #'state/config (fn [_] (prep))) :prepped)
     (throw (prep-error))))
 
-(defn- halt-system [system]
-  (when system (ig/halt! system)))
+(defn all-system-keys
```

Comment:
I don't think this function is necessary if `nil` is used to denote every key should be used.